inc/taosws_helpers.h

```diff
@@ -49,7 +49,7 @@
   WS_RES *__res  = res;                                              \
   int _e         = ws_errno(__res);                                  \
   const char *_s = ws_errstr(__res);                                 \
-  if (!_s || !*_s) break;                                            \
+  if (!_e) break;                                            \
```

Comment:
this was intentional at the very beginning, no matter whether _e is zero or not, we'll dump ws_errstr(__res) if it returns non-empty string.

Reply:
Okay, I'll restore to the original use,